README.md

```diff
@@ -1,85 +1,141 @@
-* This is helpful when building images to be called as the heavy-lifting component of an ETL.
-* This is not a part of building or operating the stack locally.
+# DTS Airflow stack
 
-```
-docker buildx build \
---push \
---platform linux/arm/v7,linux/arm64/v8,linux/amd64 \
---tag frankinaustin/signal-annotate:latest .
-```
+This stack is used to run DTS ETL processes and the production instance is deployed on `atd-data03`. Local development is available, and instructions are below.
+
+The stack is composed of:
+
+- Airflow v2 ([Docker image](https://hub.docker.com/r/apache/airflow))
+- [HAProxy](https://www.haproxy.org/) to distribute HTTP requests over the stack
+- [Flower](https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/security/flower.html) workers dashboard to monitor remote workers
+- Webhook to trigger git pulls using the [smee.io client](https://github.com/probot/smee-client)
+
+## Getting Started
+
+### Local Setup
 
-## Local Setup
-* Create, or get from a fellow dev, a `.env` file:
+To get started, create a `.env` file with the following variables:
 
 ```
 AIRFLOW_UID=0
-ENVIRONMENT=<development|production>
+ENVIRONMENT=development
 _AIRFLOW_WWW_USER_USERNAME=admin
-_AIRFLOW_WWW_USER_PASSWORD=<pick your initial admin pw here>
-AIRFLOW_PROJ_DIR=<absolute path of airflow checkout>
-OP_API_TOKEN=<Get from 1Password here: 'name TBD'>
-OP_CONNECT=<URL of the 1Password Connect install>
-OP_VAULT_ID=<OP Vault ID>
+_AIRFLOW_WWW_USER_PASSWORD=<Pick your initial admin password here>
+AIRFLOW_PROJ_DIR=<The absolute path of your Airflow repository checkout>
+OP_API_TOKEN=<Get from 1Password entry named "Connect Server: Production Access Token: API Accessible Secrets">
+OP_CONNECT=<Get from 1Password entry named "Endpoint for 1Password Connect Server API">
+OP_VAULT_ID=<Get from 1Password entry named "Vault ID of API Accessible Secrets vault">
 ```
 
-* `docker compose build`
-* `docker compose up -d`
-* Airflow is available at http://localhost:8080
-* The webhook flask app at http://localhost:8081
-* The workers' status page at http://localhost:8082
-* The test weather DAG output at http://localhost:8083
+Then, to build and start the stack:
 
-
-## Useful Commands
-* 🐚 get a shell on a worker, for example
-```
-docker exec -it airflow-airflow-worker-1 bash
+```bash
+$ docker compose build
+$ docker compose up -d
 ```
 
-* ⛔ Stop all containers and execute this to reset your local database.
-  * Do not run in production unless you feel really great about your backups. 
-  * This will reset the history of your dag runs and switch states.
+Now,
+
+- Airflow is available at http://localhost:8080
+  - You can log in to the dashboard using the username and password set in your `.env` file
+- The webhook flask app is available at http://localhost:8081
+- The workers' status page available at http://localhost:8082 
+- The test weather DAG output is available at http://localhost:8083
+
+### Developing a DAG
```

Comment:
i feel like this could be called "Executing a DAG"? "developing" makes me think of the creation of the DAG in the first place. nitpicky and i dont care too much either way so this doesnt have to change :p

Reply:
Not nitpicky at all! this section was intended to say where to put files, utils, etc for them to be usable, but I think that adding some details about how to run the DAGs and how that relates to the dashboard would be good too. 🙏 I'll roll that into this section and update the title.